src/lsp_data.rs

```diff
@@ -80,6 +87,16 @@ pub mod ls_util {
         }
     }
 
+    pub fn location_from_racer_match(mtch: racer::Match) -> Option<Location> {
+        let source_path = &mtch.filepath;
+
+        if let Some(racer::Coordinate { line, column }) = mtch.coords {
+            Some(location_from_position(source_path, line - 1, column))
+        } else {
+            None
+        }
+    }
+
```

Comment:
I think the two functions you've added to this module should be in their own module because they are not just dealing with LSP stuff. Or you could put them in actions.rs, since I think that is the only place they are used.